build.gradle

```diff
@@ -110,16 +109,22 @@ repositories {
 
 // Primary dependencies definition
 dependencies {
-    compile group: 'org.slf4j', name: 'slf4j-api', version: '1.7.7'
+    compile group: 'org.slf4j', name: 'slf4j-api', version: '2.0.0-alpha1'
```

Comment:
why are you using _alpha_ releases of these dependencies? Do they bring any advantages or notable improvements? 

If possible, we should use stable release of our dependencies to avoid trouble in the future...

Reply:
They haven't release with modules support.
Then use alpha1. It's works :3